pkg/controller/controller.go

```diff
@@ -40,6 +40,7 @@ var (
 	supportedPVProvisioners = map[string]struct{}{
 		"kubernetes.io/gce-pd":  {},
 		"kubernetes.io/aws-ebs": {},
+		"none":                  {},
```

Comment:
if user sets this to none, a cluster with backup PV enabled should fail on startup with a good failure reason. have you tried that?